src/main/java/liuni/resources/LiUniResource.java

```diff
@@ -115,11 +114,19 @@ public Response postTweet(@FormParam("message") String message) {
             logger.error("Produced an error with a " + error.getErrorStatus() + " code.", e);
 
             return Response.noContent()
-                           .type(MediaType.APPLICATION_JSON)
                            .status(error.getErrorStatus())
                            .entity(error)
                            .build();
         }
     }
 
+    @Override
+    public void filter(ContainerRequestContext request, ContainerResponseContext response) throws IOException {
```

Comment:
is there a more generic solution? i.e. if you needed to create other resource files, it would work for that too?